examples/mixture/plot_concentration_prior.py

```diff
@@ -103,7 +103,7 @@ def plot_results(ax1, ax2, estimator, X, y, title, plot_title=False):
 # mean_precision_prior= 0.8 to minimize the influence of the prior
 estimators = [
     (
-        "Finite mixture with a Dirichlet distribution\nprior and " r"$\gamma_0=$",
+        "Finite mixture with a Dirichlet distribution\nprior and $\\gamma_0=$",
```

Comment:
raw strings are easier to understand. Rather keep them than escaping characters.

Reply:
Sure I'll revert this, or perhaps change to split on `\n` which I find more readable:

```diff
-        "Finite mixture with a Dirichlet distribution\nprior and " r"$\gamma_0=$",
+        "Finite mixture with a Dirichlet distribution\n" r"prior and $\gamma_0=$",
```